### ISSUE TITLE
Revert ACTIVE jobs upon load to INACTIVE

### ISSUE THREAD

**Issue by hajnalben** [hajipy/embedded-queue#7]
I have experienced a minor issue when my process using embedded-queue is killed unexpectedly the currently active (unfinished) jobs won't be processed again upon restarting the program.

The flow:
1. client adds a new job (queue.createJob)
2. the job is written to the nedb as INACTIVE
3. client registers a processor (queue.process)
4.  the job is updated at the nedb to ACTIVE
5. suddenly the program stops (I'm using it in electron so the user just kills my app)
6. the user restarts the program
7. client registers the same processor (queue.process)
8. my job created at the step 1. will stay in ACTIVE state so won't be processed again

My idea is that when loading the jobs at startup every ACTIVE job my be converted to INACTIVE (or maybe FAILED).

I could do that but this logic may be built into the library.
```javascript
const queue = await EmbeddedQueue.Queue.createQueue({
    filename: path.join(electron.remote.app.getPath('userData'), 'queue.db'),
    autoload: true,
  })

  const activeJobs = await queue.listJobs(EmbeddedQueue.State.ACTIVE)

  for (let job of activeJobs) {
    job._state = EmbeddedQueue.State.INACTIVE
    await job.update()
  }

queue.process(
   // ...
  )
```

**Comment by hajipy** [comment]
@hajnalben 
Thanks for report bug. I will fix it soon.

**Comment by hajipy** [comment]
@hajnalben 
I fixed this issue and published `v0.0.5`.
Please upgrade.

Jobs' state will be `State.FAILURE` not `State.INACTIVE` in this version.  
(Same behavior as `Queue.shutdown` time out)

I plan more flexible recovery, please see #9.
